riscv/src/compiler.rs

```diff
@@ -1428,6 +1433,40 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             ]
         }
 
+        insn if insn.starts_with("lr.w") => {
+            // Very similar to "lw":
```

Comment:
Can we somehow add an assertion about what characters come after `"lr.w"` in `insn`?

Reply:
Do you have an idea of where we can find the exhaustive list?

Reply:
Found in source: https://github.com/bminor/binutils-gdb/blob/e030ce2c79feee6a35665a6580a23dcf937ea46f/opcodes/riscv-opc.c#L332

I've enumerated all possibilities in "instruction_ends_control_flow", will that be enough?